cmd/optimizer-nri-plugin/main.go

```diff
@@ -156,11 +174,26 @@ func (p *plugin) Configure(config, runtime, version string) (stub.EventMask, err
 	return p.mask, nil
 }
 
+type PrefetchFile struct {
+	Path string
+}
+
+type CacheItem struct {
+	ImageName     string
+	ContainerName string
+	PrefetchFiles []PrefetchFile
+}
+
+type Cache struct {
+	Items map[string]*CacheItem
+}
+
 func (p *plugin) StartContainer(_ *api.PodSandbox, container *api.Container) error {
 	dir, imageName, err := GetImageName(container.Annotations)
 	if err != nil {
 		return err
 	}
+	containerName := container.Annotations[containerNameLabel]
```

Comment:
Does the annotation exist?

Reply:
Yes, even without adding annotations in container.yaml, it can still use `container.Annotations [containerNameLabel]` to get container name

Reply:
If you only need `container name`, `container.Name` is enough.